src/core/templatetags/due_dates.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from django.utils import timezone
+
+
+register = template.Library()
+
+@register.simple_tag
+def due_date(days=0, input_type="date"):
```

Comment:
This is a useful template tag and I can think of a few places where we might want to use it but it might be worth consdering a more generic name for the tag. `offset_date` or something maybe?